tests/playlists.py

```diff
@@ -684,11 +684,11 @@
 #EXTINF:10,
 #EXT-X-PROGRAM-DATE-TIME:2015-06-18T23:22:10Z
 1432451707508/ts/71737/sequence143474338.ts
-#EXT-X-CUE-OUT-CONT:CAID=0x000000002310E3A8,ElapsedTime=161,Duration=181
```

Comment:
I changed this playlist to make sure the plain `#EXT-X-CUE-OUT-CONT\n` output gets tested.